packages/session-replay-react-native/ios/NativeSessionReplay.swift

```diff
@@ -0,0 +1,149 @@
+import Foundation
+import AmplitudeSessionReplay
+
+@objc(NativeSessionReplay)
```

Comment:
I realize we haven't don't this everywhere, but let's try to prefix objc classes with `AMP`